_posts/2020-10-23-deconvolution.md

```diff
@@ -0,0 +1,446 @@
+---
+layout: post
+title: Image Analysis Redux
+tagline: Dask + CuPy + RL
+author: John Kirkham (NVIDIA) and Ben Zaitlen (NVIDIA)
+theme: twitter
+---
+{% include JB/setup %}
+
+Summary
+-------
+
+[Last year](https://blog.dask.org/2019/08/09/image-itk) we experimented with
+Dask/ITK/Scikit-Image to perform large scale image analysis on a stack of 3D
+images.  Specifically, we looked at `deconvolution`, a common method to
+_deblur_ images.  Now, a year later, we return to these experiments with a
+better understanding of how Dask and CuPy can interact, enhanced serialization
+methods, and support from the open-source community.
```

Comment:
Maybe add a list of takeaways from this post?

> ... Now, a year later, we return to these experiments with a better understanding of how Dask and CuPy can interact, enhanced serialization methods, and support from the open-source community. Specifically, this post will mention the following:
>
> 1. ...
> 2. ...

Reply:
Good idea. Added some takeaways below :)